server/controllers/auth_controller.js

```diff
@@ -139,7 +139,7 @@ updateUser = async (req, res) => {
       if (!passwordCorrect) {
         console.log("Incorrect password");
         return res.status(401).json({
-          errorMessage: "Wrong email or password provided.",
+          errorMessage: "Wrong password provided.",
```

Comment:
Can you change the errorMessage to "Wrong email or password provided" - for security reasons don't tell the user which one was wrong. Same applies for wrong email.

Reply:
but in update, we do not provide the email

Reply:
In login, it will still have the error message "Wrong email or password provided"